monai/networks/blocks/crossattention.py

```diff
@@ -62,6 +63,7 @@ def __init__(
             input_size (tuple(spatial_dim), optional): Input resolution for calculating the relative
                 positional parameter size.
             attention_dtype: cast attention operations to this dtype.
+            use_flash_attention: if True, use flash attention for a memory efficient attention mechanism.
```

Comment:
I think we should explicitly state here and elsewhere that we're using Pytorch's inbuilt flash attention implementation.

Reply:
Agreed, would be great if we could also mention the link here: https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html